client/reader/components/header/style.scss

```diff
@@ -21,6 +21,11 @@ $component-icon-padding: 6px;
 	color: var( --dashboard-header__color );
 	min-height: 65px;
 
+
+	//All the reader layout calculations are based considering the header uses a position:fixed and with high z-index
+	position: fixed;
+	z-index: 1000;
+
```

Comment:
Making changes in this file will make our upcoming migration tricky i.e. we may lose these bug fixes.

Since we are already adding `.is-reader-msd-enabled` class so I think we should update styles in `client/layout/style.scss` by using that class.

```scss
.layout.is-reader-msd-enabled {
    .dashboard-header-bar {
	  	  position: fixed;
	  	  z-index: 1000;
	  }
}
```